tests/testsuite/update.rs

```diff
@@ -1687,7 +1687,7 @@ fn update_breaking_dry_run() {
     Package::new("ws", "2.0.0").publish();
 
     p.cargo("update -Zunstable-options --dry-run --breaking")
-        .masquerade_as_nightly_cargo(&["update-breaking"])
+        .masquerade_as_nightly_cargo(&[])
```

Comment:
Why were these changed?

Reply:
Because there is no `update-breaking` feature. There's only the `-Zunstable-options`.

Reply:
We still want an identifying label on all uses of `masquerade_as_nightly_cargo` so we can easily find all tests related to an unstable feature even if there isn't an "unstable feature" for it.

Reply:
Fixed.